control_protocol.md

```diff
@@ -100,34 +59,34 @@ A message consists of 4 or more frames.
 #### Directory
 
 Each Coordinator shall have a list of the Components (including other Connectors) connected to it.
-This is its _local Directory_.
+This is its _Directory_.
 
-The _global Directory_ is the combination of the local directories of all Coordinators in a Network.
+The _Glossary_ is the combination of the Directories of all Coordinators in a Network.
```

Comment:
?? :confused:
Ups, what I meant with my comment is: "should 'global/local directory' get an entry in the glossary?"

Reply:
@bklebel I misunderstood bilderbuchis comment (in another place), where he wanted me to put these names in the glossary, and changed the name to glossary.

I'm going to revert to local/global directory.

Reply:
Done

Reply:
Thanks :relieved: